### PR TITLE
Fix for #387

### PR DESCRIPTION
**Pull request by SujathaSIyer**
#387 
This will fix the issue of training getting killed after 1 epoch. The model is wrapped with DistributedDataParallel. Hence the attributes can be accessed by model.module.<attr name>

### CODE REVIEW

**Comment by jybaek** [comment]
This is not a good way. I think you should fix it by referring to the link [here](https://github.com/SeanNaren/deepspeech.pytorch/commit/7c8e6237904a9adf6fb15b7be2d5aea1973fedd1#diff-93ff136fae812392eb0f68d1ce89b7feL259). But it does not seem to be the only place that needs to be modified, would not it be better to modify it all?

**Comment by myhugong** [comment]
I made this change.
This worked for me on 1 or 2 GPUs.    @SujathaSIyer 

`


        from apex.parallel import DistributedDataParallel

        if isinstance(model, DistributedDataParallel):
            model = model.module


        package = {
            'version': model.version,
            'hidden_size': model.hidden_size,
            'hidden_layers': model.hidden_layers,
            'rnn_type': supported_rnns_inv.get(model.rnn_type, model.rnn_type.__name__.lower()),
            'audio_conf': model.audio_conf,
            'labels': model.labels,
            'state_dict': model.state_dict(),
            'bidirectional': model.bidirectional
        }
`
